functions/model/milestone/lib.js

```diff
@@ -48,6 +48,7 @@ exports.setMilestone = function(uid, orgDomain, title, description, milestoneId,
     EndDate: endDate,
     MilestoneStatus: milestoneStatus,
     ColorCode: colorCode,
+    Deleted: false,
```

Comment:
rather than adding new field use MilestoneStatus and update it to deleted

Reply:
Done sir!, made the required changes.